object-cache.php

```diff
@@ -3,18 +3,51 @@
 /*
 Plugin Name: Memcached
 Description: Memcached backend for the WP Object Cache.
-Version: 2.0.2
 Plugin URI: http://wordpress.org/extend/plugins/memcached/
 Author: Ryan Boren, Denis de Bernardy, Matt Martz
 
 Install this file to wp-content/object-cache.php
 */
 
-// Users with setups where multiple installs share a common wp-config.php or $table_prefix
-// can use this to guarantee uniqueness for the keys generated by this object cache
-if ( !defined( 'WP_CACHE_KEY_SALT' ) ) {
-	define( 'WP_CACHE_KEY_SALT', DB_NAME );
-}
+/*
+ * Users with setups where multiple installs share a common wp-config.php or $table_prefix
+ * can use this to guarantee uniqueness for the keys generated by this object cache
+ */
+defined( 'WP_CACHE_KEY_SALT' ) or define( 'WP_CACHE_KEY_SALT', DB_NAME );
+
+/**
+ * Should the connection to memcache be persistent (default true)
+ */
+defined( 'WP_MEMCACHE_PERSISTENT' ) or define( 'WP_MEMCACHE_PERSISTENT', true );
+
+/**
+ * Value in seconds which will be used for connecting to the daemon. (default 1)
+ */
+defined( 'WP_MEMCACHE_TIMEOUT' ) or define( 'WP_MEMCACHE_TIMEOUT', 1 );
+
+/**
+ * Number of buckets to create for this server which in turn control its probability
+ * of it being selected. The probability is relative to the total weight of all servers.
+ * (default 1)
+ */
+defined( 'WP_MEMCACHE_WEIGHT' ) or define( 'WP_MEMCACHE_WEIGHT', 1 );
+
+/**
+ * Controls how often a failed server will be retried, the default value is 15 seconds
+ */
+defined( 'WP_MEMCACHE_RETRY' ) or define( 'WP_MEMCACHE_RETRY', 15 );
+
+/**
+ * Controls if memcache writes to log
+ */
+defined( 'WP_MEMCACHE_LOGGING' ) or define( 'WP_MEMCACHE_LOGGING', false );
+
+/**
+ * Disable / Enable memcache flushing
+ */
+defined( 'WP_MEMCACHE_FLUSHING' ) or define( 'WP_MEMCACHE_FLUSHING', true );
```

Comment:
This should have the action in the name, otherwise it's not clear if this is saying "memcached is currently flushing". +1 for `WP_MEMCACHE_DISABLE_FLUSHING`